src/scss/vendors/_vendors.scss

```diff
@@ -1 +1,70 @@
 // Vendor-specific styles.
+
+//UIO
+
+//toc
+.flc-toc-tocContainer {
+	background-color: $white;
+	padding-top: rem(39.84);
+
+	.flc-toc-header {
+		margin-top: 0;
+	}
+}
+
+.flc-toc-tocContainer * {
+	margin: $white;
+}
+
+//uio tab
+#defaultContainer .fl-prefsEditor-separatedPanel .fl-panelBar .fl-prefsEditor-buttons {
+	background: $white;
+	border: 1px solid $green-dark;
+	border-radius: 0 0 rem(30) rem(30);
+	border-top: 1px solid $white;
+	box-shadow: none;
+	box-sizing: border-box;
+	padding: rem(10) rem(20);
+
+	button {
+		color: $black;
+		font-family: Roboto;
+		font-size: rem(16);
+		font-style: normal;
+		font-weight: $font-weight-normal;
+		line-height: rem(19);
+	}
+}
+
+//focus
+.fl-theme-bw {
+	@include uio-focus-style($black);
```

Comment:
It would be better to use actual color code from infusion: https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/css/stylus/utils/Themes.styl#L4-L55